tests/conn_can/README.md

```diff
@@ -78,6 +78,13 @@ To send a raw CAN frame, id 0x100 with 2 bytes of data 01 02 on interface 0:
 test_can send 0 100 01 02
 ```
 
+To send a raw CAN remote request frame, id 0x100 on interface 0 and datalenght 7:
+```
+test_can sendrtr 0 100 7
+```
+A remote request frame has a NULL payload but can have a specific DLC (0 to 8).
```

Comment:
What is a DLC?

Reply:
data length code (DLC). I added it to Readme